src/main/java/com/minecolonies/coremod/colony/requestsystem/management/handlers/RequestHandler.java

```diff
@@ -498,15 +498,15 @@ public static IRequest getRequestOrNull(final IStandardRequestManager manager, f
     private final static class AssigningResult implements Comparable<AssigningResult>
     {
         private final IRequestResolver resolver;
-        private final List<IToken>     children;
+        private final List<IToken<?>>     children;
 
-        private AssigningResult(final IRequestResolver resolver, final List<IToken> children)
+        private AssigningResult(final IRequestResolver resolver, final List<IToken<?>> children)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "AssigningResult" constructor. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AUnusedPrivateMethod)